anta/tests/connectivity.py

```diff
@@ -33,16 +33,24 @@ class VerifyReachability(AntaTest):
             - source: Management0
               destination: 1.1.1.1
               vrf: MGMT
+              df_bit: True
+              size: 100
```

Comment:
Can you also update `anta/examples/tests.yaml` with these add-ons?

Reply:
Added these add-ons to` anta/examples/tests.yaml`
Thanks!